pinot-core/src/main/java/com/linkedin/pinot/core/operator/BaseOperator.java

```diff
@@ -29,7 +29,7 @@
 public abstract class BaseOperator implements Operator {
   private static final Logger LOGGER = LoggerFactory.getLogger(BaseOperator.class);
 
-  private final String _operatorName = getClass().getSimpleName();
+  private final String _operatorName = getOperatorName();
```

Comment:
Will address it in separate PR, this one was already merged before your comment.